text-utilities/text.html

```diff
@@ -77,6 +77,18 @@ <h2 class="p-1">Text Alignment</h2>
             sandbox="allow-scripts allow-same-origin">
           </iframe>
            </div>
+           <h2 class="p-1">Simple text utilities</h2>
+           <div class="card p-1 d-flex justify-around">
+                <div class="text-grey">Grey Text</div>
+               <div class="text-small">small Text</div>
+           </div>
+           <div class="doc-img">
+            <iframe
+            src="https://carbon.now.sh/embed?bg=rgba%28171%2C+184%2C+195%2C+1%29&t=seti&wt=none&l=auto&width=680&ds=true&dsyoff=20px&dsblur=68px&wc=true&wa=true&pv=56px&ph=56px&ln=false&fl=1&fm=Hack&fs=14px&lh=133%25&si=false&es=2x&wm=false&code=%253Cdiv%2520class%253D%2522text-grey%2522%253EGrey%2520Text%253C%252Fdiv%253E%250A%2520%253Cdiv%2520class%253D%2522text-small%2522%253Esmall%2520Text%253C%252Fdiv%253E%250A"
+            style="width: 100%; height: 242px; border:0; transform: scale(1); overflow:hidden;margin: 1rem auto;"
+            sandbox="allow-scripts allow-same-origin">
```

Comment:
For iframe give your own css style. Remove this default style. Default style is not responsive.

Reply:
Okay